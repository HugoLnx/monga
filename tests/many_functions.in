int xpto;
char[] hey;
void main() {
}

int func() {
}
