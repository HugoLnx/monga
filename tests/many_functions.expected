declarations:
  declaration:
    variables:259,0
      variable:xpto
  declaration:
    variables:260,1
      variable:hey
  declaration:
    function:main,262,0
      block:
  declaration:
    function:func,259,0
      block:
